worktree_status.go

```diff
@@ -370,7 +370,11 @@ func (w *Worktree) doAdd(path string, ignorePattern []gitignore.Pattern, skipSta
 		}
 	}
 
+	// Relative paths only; see: https://github.com/go-git/go-git/issues/1460
 	path = filepath.Clean(path)
+	if filepath.IsAbs(path) {
+		path, _ = filepath.Rel(w.Filesystem.Root(), path)
+	}
```

Comment:
I believe this is not compatible with [BoundOS](https://github.com/go-git/go-billy/blob/main/osfs/os_bound.go#L48), and unfortunately `go-billy` does not expose an os-aware Rel func.

Reply:
Correct, it fails spectacularly under BoundOS.  Yikes.

But `Rel` seems to work OK, I'll push another attempt at a fix for your consideration.